script.js

```diff
@@ -0,0 +1,67 @@
+$(document).ready(function() {
+
+  var square = $(".square");
+  var reset = $("#reset");
+  var winner = $("#winner");
+  var board = $("#board");
+  var clickCounter = 0;
+  var xOwned = [];
+  var oOwned = [];
+  var winners = [
+    ["1", "2", "3"],
+    ["4", "5", "6"],
+    ["7", "8", "9"],
+    ["1", "4", "7"],
+    ["2", "5", "8"],
+    ["3", "6", "9"],
+    ["1", "5", "9"],
+    ["7", "8", "9"]
+  ];
```

Comment:
:+1: Good effort with the mathematical solution! I appreciate that you changed it up to get it functioning :)